addons/common/functions/fnc_exportMissionSQF.sqf

```diff
@@ -114,6 +114,7 @@ private _fnc_processInventory = {
 
     if (_object isKindOf "CAManBase") then {
         _outputObjects pushBack ["%1 setUnitLoadout %2;", _varName, getUnitLoadout _object];
+        _outputObjects pushBack ["%1 call BIN_fnc_CBRNHoseInit;", _varName];
```

Comment:
The incorrect facemask is returned by `getUnitLoadout`, leaving the unit without a respirator when the SQF is executed but the hose still exists.

Reply:
Maybe a frame delay will fix this issue?

Reply:
Checking again the mask appears for one frame and is then removed for some reason.

Reply:
Yes, `BIN_fnc_CBRNHoseInit` and `setUnitLoadout` needs to be executed in the next frame. Dunno how I missed it.

Reply:
Should be resolved now